libs/perun/services/src/lib/init-auth.service.ts

```diff
@@ -111,9 +111,17 @@ export class InitAuthService {
       );
     } else {
       this.setLoginScreen(true);
+      const query = location.search.substr(1).split('&');
+      let queryParams = null;
+      for(const param of query) {
+        const p = param.split('=');
+        if(p[0] === 'lang'){
+          queryParams = {lang: p[1]};
+        }
+      }
```

Comment:
I think we want all the queryParams to be redirected to login page, not just lang